selfdrive/boardd/pandad.py

```diff
@@ -125,6 +126,16 @@ def main() -> NoReturn:
       # log panda fw versions
       params.put("PandaSignatures", b','.join(p.get_signature() for p in pandas))
 
+      # store last panda logs
+      LOG_AMOUNT = 100
+      logs = {}
+      for panda in pandas:
+        try:
+          logs[panda.get_usb_serial()] = panda.get_logs(True)[-LOG_AMOUNT:]
+        except Exception:
+          cloudlog.exception(f"Error getting logs for {panda.get_usb_serial()}")
+      params.put("PandaLogs", json.dumps(logs, default=str))
```

Comment:
the param doesn't seem quite right since you don't get the logs if you don't go onroad. why not just cloudlog? would also add a log if it exceeds 100